source/agora/consensus/Quorum.d

```diff
@@ -479,13 +502,14 @@ private NodeStake[] buildStakesDescending (const ref PublicKey filter,
 *******************************************************************************/
 
 version (unittest)
-private QuorumConfig[PublicKey] buildTestQuorums (Range)(Range amounts,
+private QuorumConfig[Hash] buildTestQuorums (Range)(Range amounts,
     const(PublicKey)[] keys, const auto ref Hash rand_seed,
-    const auto ref QuorumParams params, int id)
+    const auto ref QuorumParams params, int id, ref PublicKey[Hash] utxo_to_key)
```

Comment:
Can't you just pass `const(Hash)[] keys` instead of `const(PublicKey)[]` and remove `utxo_to_key` ??

Reply:
Perhaps not, since we generates txs from within this function. But the AA by ref is a red flag.

Reply:
Will apply `out` instead of `ref`.